### ISSUE TITLE
В поле поиска лекарств введенные данные не удаляются после отправки. 

### ISSUE THREAD

**Issue by Yamortis** [Yamortis/Test_health.mail.ru#4]
**Предусловие:**
1. Открыть в браузере страницу сервиса https://health.mail.ru/
2. Скроллом или бегунком пролистать страницу вниз до блока "Лекарства" https://yadi.sk/i/hEa6qed-vDf_nA

**Шаги:**
1. В поле для поиска лекарств ввести название лекарства (например, Лекролин) https://yadi.sk/i/zY57Y3nNz10IfA и нажать на лупу или кнопку Enter на клавиатуре. Откроется страница с результатами поиска по введенному запросу.
2. Нажать кнопку "Назад" в браузере.

**Фактический результат:**
в поле остается введенный ранее запрос. Скринкаст https://yadi.sk/i/bFm9eOAylkUzcg

**Ожидаемый результат:**
поле для поиска лекарств пустое.

**Comment by Svetlana678** [comment]
Да, ошибка, всё воспроизводится. Введенный запрос в поле поиска лекарств исчезает только после обновления страницы.
